src/utilities/connectors.ts

```diff
@@ -12,9 +12,10 @@ const chainId = parseInt(process.env.REACT_APP_CHAIN_ID || '97', 10);
 export const injected = new InjectedConnector({ supportedChainIds: [chainId] });
 
 const walletconnect = new WalletConnectConnector({
-  rpc: { [chainId]: rpcUrl },
+  rpc: { 56: rpcUrl },
```

Comment:
Shouldn't we move such things to constants/config builder functions?

Reply:
Yeah definitely. Here too, sorry for the confusion, since this is a hotfix I went the other way. The draft PR I opened to add this fix to `develop` uses the config :)
https://github.com/VenusProtocol/venus-protocol-interface/pull/269